### DISCUSSION TITLE
Question. Can I see id(primary key) in the view?

### DISCUSSION THREAD

**Discussion by Charleslee522** [nocodb/nocodb#611] [Q&A] [answered]
**Please enter the following details**

Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **sqlite3**
ProjectOnRootDB: **true**
RootDB: **sqlite3**
PackageVersion: **0.11.42**

**Expected behavior**

I think it's not a bug or something. I just have a question.
In the view there is no `id(primary key)`. Is there any option to view id(pk)?
There is only `#`(The sequence in this page I guess. It's different with id.)
I think #177 is the same issue with it.

**Screenshots**

<img width="901" alt="스크린샷 2021-10-05 오전 8 55 18" src="https://user-images.githubusercontent.com/15343839/135939782-e0cbaef0-41dc-4f63-8e20-a78f7f7cd0fc.png">

**Comment by wingkwong** [comment, marked as answer]
Hi @Charleslee522 

``id`` is a system field, which you can enable it in the following steps:

1. Click ``Fields``

2. Tick ``Show System Fields``

![image](https://user-images.githubusercontent.com/35857179/135953833-a7279186-8263-448c-98ea-7a845cff5c64.png)

3. Tick the system fields that you want to show. In your case, that is ``id``. 

![image](https://user-images.githubusercontent.com/35857179/135953846-92ad2f60-1846-47e5-87fe-e26a7cb7aa53.png)

4. You should see it s shown in your grid view

![image](https://user-images.githubusercontent.com/35857179/135953807-c2b81330-c848-4820-b236-a2e181f68b75.png)

**Comment by o1lab** [comment]
Moving to discussions
